en/docs/developer-guides/protocols-and-connectors/supported-protocols.md

```diff
@@ -0,0 +1,19 @@
+
+| Protocol | Description                                                                                                                                 | Documentation                                                                                                                                                                                                                                                                                                                                                                                                  |
+|-------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <a href="https://central.ballerina.io/ballerina/http" target="_blank">HTTP</a>                          | HTTP client/server functionalities to produce and consume HTTP APIs                                                                         | <ul> <li> <a href="https://lib.ballerina.io/ballerina/http/latest#Client" target="_blank">HTTP client</a><br> </li> <li> <a href="https://lib.ballerina.io/ballerina/http/latest#Listener" target="_blank">HTTP listener</a> </li> </ul>                                                                                                                                                              |
```

Comment:
Avoid url redirection. Use central.ballerina.io urls instead of lib.ballerna.io urls